DiceKit/Die.swift

```diff
@@ -116,3 +116,14 @@ extension Die: ExpressionType {
 public func == (lhs: Die, rhs: Die) -> Bool {
     return lhs.sides == rhs.sides
 }
+
+// MARK: - Debug
+extension Die: CustomDebugStringConvertible {
+    
+    public var debugDescription: String {
+        get {
+            return "I am a Die! I have \(self.sides) sides!"
```

Comment:
Let's also look into `print` versus `debugPrint`. I think `print` should show `d6` and `8`, while `debugPrint` should show `Die(sides: 6)` and `Constant(8)`?